### DISCUSSION TITLE
metaSPAdes or metaviralSPAdes for viral contigs assembly?

### DISCUSSION THREAD

**Discussion by ShailNair** [ablab/spades#979] [Q&A]
Hi,

 I have been using metaSPAdes for bacterial assembly. Recently we used metaSPAdes to assemble contigs from fastq files in one of our projects focusing on algae-associated microbiome exploration. Now, we want to assemble viral contigs from the same samples. 

As I [understand](https://academic.oup.com/bioinformatics/article/36/14/4126/5837667?login=true#211359400) metaviralSPAdes outperforms metaSPAdes in assembling complete viral contigs. Does this mean that metaSPAdes is still superior in obtaining all viral contigs (complete and near-complete)?

**Comment by asl** [comment]
metaSPAdes and metaviralSPAdes have different aims. The latter tries to reconstruct more complete viral genomes (but additional checks are required afterwards). metaSPAdes are for the whole metagenome assemblies.

Hope this helps

**Reply by ShailNair** [reply]
Thanks
